README.md

```diff
@@ -11,6 +11,23 @@ variables. You will need to set F90 (and optionally, F90FLAGS).
 At link time you will need to link with the OpenCL run-time
 libraries on your system.
 
+# Controlling runtime behaviour"
+
+There are 3 environment variables to control the OpenCL execution
+at runtime:
+
+- `FORTCL_KERNELS_FILE`: Allows to specify at run-time the binary or
+source code filename containing the OpenCL kernels to execute.
+
+- `FORTCL_PLATFORM`: Integer that selects in which OpenCL platform the
+kernels are going to be executed (the list of OpenCL platforms can be
+queried with the `clinfo` command). If this variable is not set, the
+kernels will be launched on platform 1.
```

Comment:
Please extend to clarify whether this list is indexed from 0 or 1. e.g "...on platform 1 (the first in the list)."

Reply:
Done